JSDemos/Demos/Charts/Candlestick/description.md

```diff
@@ -1 +1,10 @@
+
+In Candlestick series, each point consists of a rectangle (body) and a vertical line (shadow, wick or tail). The top and bottom values of a vertical line correspond to the highest and lowest prices of the stock. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for these prices.
+
+The bottom and topmost values of the point's body correspond to the opening and closing price of a stock. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties. If the stock closes higher than its opening price, no color fills the body and if the stock closes lower than its opening price, the Chart applies a color fill to the body.
+
+To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To change color of other points, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[color](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/#color) property; you can also specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[innerColor](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#innerColor) property to fill these points' bodies with color. To switch between high, low, open or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
+
+Additionally, use the [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) property to declare a custom tooltip that shows prices. You can also enable the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/).[workdaysOnly](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#workdaysOnly) property to display only the work days on the argument axis.
```

Comment:
In this paragraph, you just need to mention a few additional options. You try to make it look like a continuous narrative by adding transitional phrases like "Additionally" or "You can also". But there's really no flow to these statements - they are separate from each other. I'd rather just use a list with a short introduction: 

```suggestion
The following settings may help you make your chart more user-friendly:

- [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) - specifies a  custom tooltip template so you can display additional information, such as price values. 
- [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/).[workdaysOnly](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#workdaysOnly) - displays only work days on the argument axis.
```